coresimd/src/x86/i586/abm.rs

```diff
@@ -44,16 +44,16 @@ pub unsafe fn _lzcnt_u64(x: u64) -> u64 {
 #[inline(always)]
 #[target_feature = "+popcnt"]
 #[cfg_attr(test, assert_instr(popcnt))]
-pub unsafe fn _popcnt32(x: u32) -> u32 {
-    x.count_ones()
```

Comment:
This was on purpose, bit manipulation intrinsics prefer unsigned integers like the ones in `std::` do. Whatever we decide to do, we should choose a guideline, and follow it consistently (many other intrinsics that take `__m128i` take for example `u32x4` instead of `i32x4`).

We should open an issue for this.